index.js

```diff
@@ -0,0 +1,95 @@
+// Storage Functions
+const getBooks = () => {
```

Comment:
- [ ] Please 🙏🏽 create a collection that keeps a list of books (hint: you can use an array of objects for that) to populate your UI at the moment it opens.